scripts/sv/fermiLite/scanBam.sh

```diff
@@ -22,13 +22,15 @@ REF_FASTA="${MASTER_NODE}$5"
 REF_INDEX_IMAGE="$6"
 INTERVAL_KILL_LIST=$(echo "${REF_FASTA}" | sed 's/.fasta$/.kill.intervals/')
 KMER_KILL_LIST=$(echo "${REF_FASTA}" | sed 's/.fasta$/.kill.kmers/')
+ALTS_KILL_LIST=$(echo "${REF_FASTA}" | sed 's/.fasta$/.kill.alts/')
```

Comment:
The file /seq/references/Homo_sapiens_assembly38/v0/Homo_sapiens_assembly38.fasta.64.alt is missing 170 alt contigs that are in your kill list, though.